src/ggrc/models/document.py

```
@@ -26,8 +26,11 @@ class Document(Ownable, Relatable, Base, Indexed, db.Model):
   year_id = deferred(db.Column(db.Integer), 'Document')
   language_id = deferred(db.Column(db.Integer), 'Document')
 
-  object_documents = db.relationship(
-      'ObjectDocument', backref='document', cascade='all, delete-orphan')
+  URL = 1
+  ATTACHMENT = 2
+  document_type = deferred(db.Column(db.Integer, default=URL, nullable=False),
+                           'Document')
```

A couple of things here:

1. Attachment should be the default here as it is a lot more common throughout the app.
2. There is no validation so the frontend guys can easily send document_url = 0 or document_url = 3 and we are going to store that in the database without blinking. We should throw an error in this case.
3. Defining URL = 1, ATTACHMENT = 2 is not a good idea since this data needs to make it down to the frontend somehow and if we just define the same constants on the frontend we might get a bug if they change.

I propose you either change the column into a db.Boolean named `is_url` or a db.Enum to make sure we don't get corrupt data into the database.